ckanext/unaids/theme/templates/package/releases/list.html

```diff
@@ -17,25 +17,21 @@
         </p>
     {% else %}
         <div class="btn-group">
-            <a href="{{ urls.create_from_latest_version }}" class="btn btn-danger">
-                <span>{{ _('Add Release') }}</span>
+            <a href="{{ urls.create_from_latest_version }}" class="btn btn-primary">
+                <i class="fa fa-plus-square"></i> <span>{{ _('Add Release') }}</span>
```

Comment:
I have added the plus sign here myself as we use it for all other "Add" buttons.